### DISCUSSION TITLE
Possible change between networks without wallet connection

### DISCUSSION THREAD

**Discussion by internetmoneydev** [rainbow-me/rainbowkit#1208] [Q&A]
hi there, is it possible to flip between networks without requiring the user to connect their wallet? i am starting to look into this and it seems like it should be a pretty standard configuration but i don't see any way to do it. it looks like i am going to have to flip controls between my useState and rainbowkit's does anyone have a simple example that they have used in the past perhaps? doesn't seems straightforward to me

**Comment by internetmoneydev** [comment]
it appears this is not available due to this line: https://github.com/rainbow-me/rainbowkit/blob/737a1d6ee13f409fc2060fa55d6c741a7fa574db/packages/rainbowkit/src/components/RainbowKitProvider/ModalContext.tsx#L99

**Comment by Markcode1992** [comment]
Also following

**Comment by 0xmkh** [comment]
@internetmoneydev This should be fixed in the RainbowKit v2 launch
